spec/controllers/article_spec.rb

```diff
@@ -1,24 +1,20 @@
 require 'rails_helper'
+require 'spec_helper'
 
 RSpec.describe ArticlesController do
   describe 'GET #index' do
     it 'responds successfully with an HTTP 200 status code' do
+      @user = User.create(:email    => "imanexample@example.com",:password => "shhhpassword")
```

Comment:
- I think you are not running rubocop in specs
- I recommend you using FactoryGirl gem